modules/iam-user/outputs.tf

```diff
@@ -57,11 +57,11 @@ output "this_iam_access_key_encrypted_secret" {
 }
 
 output "this_iam_access_key_ses_smtp_password" {
-  description = "The secret access key converted into an SES SMTP password"
+  description = "The secret access key converted into an SES SMTP password by applying AWS's Sigv4 conversion algorithm."
```

Comment:
Please add another output instead of replacing it.

Reply:
Alright, thanks for quick review!

Reply:
Done. Updated README as well.